### ISSUE TITLE
No Inhabited Drak Systems

### ISSUE THREAD

**Issue by McloughlinGuy** [endless-sky/endless-sky#5532]
**Is your feature request related to a problem? Please describe.**
It is mentioned by the Quarg that the Drak live nearby. But the only system marked as the Drak government is inhabitted by a single Archon.

**Related Issue Links**
I looked, and I found nothing like this except someone suggesting something in the second galaxy, which is different than what I'm asking.

**Describe the solution you'd like**
I would like there to be a drak system in the milky way.

**Describe alternatives you've considered**
An alternative would be that someone tell me why there are no inhabited Drak systems.

**Additional context**
I would like to see the Drak more or at least know a little more about them,.

**Comment by MCOfficer** [comment]
I'm not sure how official it is, but lore-wise, the drak basically ascended into another dimension, and thus don't need material forms (think "Interstellar"). Thus it wouldn't make sense for them to inhabit physical worlds. When the Quarg mention that they are nearby, I don't believe they mean physically nearby - it's more of an "they are watching".

**Comment by Zitchas** [comment]
There is also the possibility that their worlds - perhaps through the inter-dimensionality that MCO mentioned - are or at least were nearby, but have something unusual about them that means that Jump Drives can't reach them - at least, not how we know how to use them. 
- Maybe they are partially out-of-phase with our universe? 
- Maybe they are hidden somehow? 
- Maybe one of the system we can visit *is* the Drak system, but their planet is cloaked or not in our dimension anymore. 
- Maybe the Ember Waste is the result of a Drak war, and all those systems used to be the heart of their empire, but were largely destroyed and all the current existing Drak are extra dimensional, or permacloaked, or something.
- Maybe their stars have expired or been consumed somehow, and thus there is no gravity well or other "system" identifier for our navigational system to home in on?
- Maybe the Drak evolved in a stellar nursery gas cloud, and so there isn't a single point that our nav systems can home in on, but in theory we might be able to get to later on if we have precise instructions on how to get there?
- Maybe the Archons are actually Drak world-ships, and the Drak themselves are either microscopic creatures, or have transcended needing a physical form, and thus each Archon is "inhabited" by billions or trillions of Drak and the personality that talks to us might be the ship itself... Or it might be the collective consciousness of all the Drak on board. 

In short, there are a *lot* of ways the Drak could be "nearby" without actually having a planet that we can visit or a system that we can find.

**Comment by McloughlinGuy** [comment]
This issue isn't going anywhere, I'm closing it.
